### ISSUE TITLE
Checklist for final demonstration ⚠️

### ISSUE THREAD

**Issue by eldenpark** [contact-tracer-csci-526/contact-tracer#135]
To streamline the communication, you are encouraged to discuss in our Slack workspace. Though you can still talk about each item on this page, try as much to use here to list the items.

Edit the following list if necessary.

## Todos
- [x] Make balls wait until a user draws circle in the center for tutorials [@Bhagyashree-Rawal ]
- [x] Add contents to document (README.md) [all]
- [x] Reduce line length [@RuchaTambe]
- [x] Port to mobile devices [@RuchaTambe @eldeni @JBeavans ]
- [x] Port to web [@eldeni]
- [x] ~~Fix Gray edge along the contour of the game scene (device size and gameScene ratio)~~ [@jungwony629 ]
- [x] Reduce size of normal balls [@patrick-assaf]
- [x] Change expected score [@patrick-assaf]
- [x] Collider remains even after line disappears [@eldeni ]
- [x] Add `Year2020` level [@eldeni ]
- [x] Fix ball walking along the wall [@jungwony629 ]
- [x] Change position of pause button [@pspatel04 ]
- [x] Add skip button to the tutorial  [ @pspatel04 ]
- [x] Add `Year2020` game level main menu [ @pspatel04 ]
- [x] Replace 'menu' button with two buttons- tutorial and game  [ @pspatel04 ]
- [x] Change the hand sprite [@patrick-assaf]
- [x] Make original virus red [@patrick-assaf]
- [x] Virus-line interaction
- [x] Increase the speed of the 'hand' in tutorial levels[@RuchaTambe]
- [x] Fix line so that it lets virus ball pass [@eldeni ]
- [x] Increase the duration of cure ball - afterall, the users are going to test on their laptops for the final [@RuchaTambe ]
- [x] Remove cure ball in level tutorial 1 and 2
- [x] Reduce wordiness of the instructions [@vkanduku1995 ]
- [x] Explain score mechanism [@vkanduku1995 ]
- [x] Add entries for different levels onto MainMenu [ @pspatel04 ]
- [x] Navigation menu for tutorial levels and main levels [ @pspatel04 ]
- [x] Change text of Tutorial "Level 1" to "Tutorial 1" [ @pspatel04 ]
- [x] Add description of cure ball for tutorial - too wordy right now - need to fix it [@vkanduku1995 ]
- [x] End game differently based on a game level
- [x] Initialize game scene differently based on a game level
- [x] Add an explanation scene before game scene differently based on a given level [@vkanduku1995 ]
- [x] Reduce the timer in tutorial levels - TLevel1: 5sec, TLevel2: 20 sec, TLevel3: 30 sec [ @pspatel04 ]
- [x] After tutorial level 1, user should navigate to pick tutorial level 2 - but not the score screen [ @pspatel04 ]
- [x] Add Story line for tutorial level 2,3 [@vkanduku1995 ]

## Suggestions (Move to Todos if you are going to take it)
- [x] There should be a timer for the frozen balls. If not, we are giving away the game for the users. Like how the users found a trick to control the virus and win the game, here they can control the game by freezing two balls
- [x] Where to explain about score?
- [x] Make the video a little professional so that we could showcase it on our LinkedIn

## Recording
- [x] July 13 (tentative) [@JBeavans ]

## Rehearsal
- [x] July 13 or 14 (tentative) [all]

**Comment by eldeni** [comment]
@vkanduku1995 How do you think about adding a small `skip` button on the tutorial storyline? That is, a user might skip the storyline if he wants.

**Comment by eldenpark** [comment]
All done. Thank you all for your ceaseless participation! Closing.
